src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditMoney.razor

```diff
@@ -1,5 +1,5 @@
-﻿@using Cryptocash.Ui.Generated.Component;
-@inherits UiEditMoneyBase
+﻿@using MudBlazor
+@inherits UiEditMoney
```

Comment:
I think we don't need this line at all, as per naming convention the EditMoney will be automatically available in this file. However the class should be partial and should have the same name as the razor part.

So for this case we should have:
EditMoney.razor
EditMoney.razor.cs

In the first file we should not have `@inherits ...`, the model will be available automatically.
But there should be defined the namespace same as in EditMoney.razor.cs:
`namespace Nox.Ui.Blazor.Lib.Components.NoxTypes`

In the second class it should be `public partial class EditMoney: ComponentBase`

@DanielHodgeGH  please test this approach and update it for all the components if it works

Reply:
thanks Anna - tested okay have added to latest commit